pkg/proxy/proxier.go

```diff
@@ -319,38 +319,43 @@ type Proxier struct {
 	hostIP        net.IP
 }
 
+var (
+	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
```

Comment:
Will add comments to the intended use (every consumer of NewProxier having the same validation error).

> On May 15, 2015, at 12:05 PM, Tim Hockin <notifications@github.com> wrote:
> 
> In pkg/proxy/proxier.go:
> 
> > @@ -319,38 +319,43 @@ type Proxier struct {
> >  	hostIP        net.IP
> >  }
> >  
> > +var (
> > +	ErrProxyOnLocalhost = fmt.Errorf("cannot proxy on localhost")
> Typed errors are fine (though it's not my favorite aspect of Go, whatevs). My point was to grok why some errors matter more than others.
> …
> —
> Reply to this email directly or view it on GitHub.
>